src/main/java/org/glovo/glovo_db/controller/OrderController.java

```diff
@@ -0,0 +1,51 @@
+package org.glovo.glovo_db.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.glovo.glovo_db.model.Order;
+import org.glovo.glovo_db.model.Product;
+import org.glovo.glovo_db.service.OrderService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+@RestController
+@RequestMapping("/orders")
+@RequiredArgsConstructor
+public class OrderController {
+    private final OrderService orderService;
+
+    @GetMapping
+    public List<Order> getAllOrders() {
+        return orderService.getAllOrders();
+    }
+
+    @GetMapping("/{id}")
+    public Order getOrderById(@PathVariable Long id) {
+        return orderService.getOrderById(id);
+    }
+
+    @PostMapping
+    public Order createOrder(@RequestBody Order order) {
+        return orderService.createOrder(order);
+    }
+
+    @PutMapping("/{id}")
+    public void updateOrder(@PathVariable Long id, @RequestBody Order updatedOrder) {
+        updatedOrder.setId(id);
+        orderService.updateOrder(updatedOrder);
+    }
+
+    @DeleteMapping("/{id}")
+    public void deleteOrder(@PathVariable Long id) {
+        orderService.deleteOrder(id);
+    }
+
+    @PostMapping("/{id}/addProduct")
```

Comment:
According to task, you need to use @Patch

Reply:
yes, thanks. check